Podcast/FeedElement.swift

```diff
@@ -35,6 +35,17 @@ enum FeedContext {
             return subject
         }
     }
+
+    var cellType: FeedElementTableViewCell.Type {
+        switch self {
+        case .followingRecommendation:
+            return FeedEpisodeTableViewCell.self
+        case .followingSubscription:
+            return FeedEpisodeTableViewCell.self
+        case .newlyReleasedEpisode:
+            return FeedEpisodeTableViewCell.self
```

Comment:
just curious, why are all three of these cases returning the same thing?

Reply:
And this is why we have code review!! Thanks for catching this